truss/templates/base.Dockerfile.jinja

```diff
@@ -2,15 +2,44 @@ ARG PYVERSION={{config.python_version}}
 FROM {{base_image_name_and_tag}}
 
 {% block fail_fast %}
-        RUN grep -w 'ID=debian\|ID_LIKE=debian' /etc/os-release || { echo "ERROR: Supplied base image is not a debian image"; exit 1; }
-        RUN which python && python --version | grep -E '3\.[0-9]|10\.[0-9][0-9]' || \
-            which python3 && python3 --version | grep -E '3\.[0-9]|10\.[0-9][0-9]' || \
-            { echo "ERROR: Supplied base image does not have 3.8 <= python <= 3.10"; exit 1; }
+RUN grep -w 'ID=debian\|ID_LIKE=debian' /etc/os-release || { echo "ERROR: Supplied base image is not a debian image"; exit 1; }
+RUN which python && python --version | grep -E '3\.[0-9]|10\.[0-9][0-9]' || \
+    which python3 && python3 --version | grep -E '3\.[0-9]|10\.[0-9][0-9]' || \
+    { echo "ERROR: Supplied base image does not have 3.8 <= python <= 3.10"; exit 1; }
 {% endblock %}
 
 # If user base image is supplied in config, apply build commands from truss base image
 {% if config.base_image %}
-    {% include truss_build_template_path %}
+ENV PYTHONUNBUFFERED True
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && \
+    apt install -y bash \
+                build-essential \
+                git \
+                curl \
+                ca-certificates \
+                software-properties-common
+
+RUN pip install --no-cache-dir --upgrade pip \
+    && rm -rf /root/.cache/pip
+
+COPY ./requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt \
```

Comment:
this should be the `base_server_requirements.txt` not `requirements.txt` which is different in this build context